Tortuga.Anchor/Tortuga.Anchor.Tests/Metadata/MetadataCacheTests.cs

```diff
@@ -926,6 +927,91 @@ public void MetadataCache_VisualBasic_Normal()
 		}
 	}
 
+
+	[TestMethod]
+	public void CloneTest_Shallow()
+	{
+		var original = new Base();
+		original.Property0 = 10;
+		original.ChildA = new() { Property = 100, PropertyA1 = 110, PropertyAX = 120 };
+		original.ChildB = new() { Property = 200, PropertyB1 = 220, PropertyBX = 230 };
+
+		var copy = MetadataCache.Clone(original, CloneOptions.None);
+		Assert.IsNotNull(original);
```

Comment:
Shouldn't we be asserting that the copy is not null here rather than the original? Or if we still want this check, we should add one for the copy as well.

Reply:
Fixed.